data/pritamrajput.json

```
@@ -21,4 +21,6 @@
       "icon": "FaGithub"
     }
   ]
```

Please add a missing comma `,` here to validate your json! 

```suggestion
  ],
```

---

Okay and than create new pull request ?

---

No-no, you did it! 🎉